### DISCUSSION TITLE
Do Retype support reusing markdown files to another markdown file?

### DISCUSSION THREAD

**Discussion by jplanas-openit** [retypeapp/retype#510] [Q&A]
We tried to use Iframes and embed component but unfortunately, retype is rendering the markdown files with headers and navigation pane.
Is there a ways we can do inclusion of external static MD files?

![image](https://user-images.githubusercontent.com/127005176/222943286-5ad63393-abea-4866-98e4-5da415b0e507.png)

Thanks in advance!

**Comment by geoffreymcgill** [comment]
I'm not sure I 100% understand your requirements. Can you explain in more detail?

Do you want to include a Retype page in another website, or do you want to include another website page inside of Retype?

Or do you want to include the content of another `.md` file inside another `.md` file?

**Reply by jplanas-openit** [reply]
Thanks for the quick reply.

The last one is the one I am achieving:
"_Or do you want to include the content of another .md file inside another .md file?_"

Because there are some md files that we need to reuse on anaother md file.

**Reply by geoffreymcgill** [reply]
The easiest technique is to add the shared `.md` file into the `_includes` folder and then within the page where you want to include that content, you use `{{ include "my-file-name" }}`. 

For example, if you had a **Warning** message you wanted to include on multiple page, create a `_includes/warning.md` file. 

```md
!!!
This is something important to highlight.
!!!
```

Then on any page within your project, use the following syntax to include the Warning. 

```md
{{ include "warning" }}
```

The included file doesn't need to be in the `_include` folder, but you would need to add the full path and file extension if it isn't. For example:

```md
{{ include "/helpers/warning.md" }}
```

Hope this helps.

**Reply by jplanas-openit** [reply]
Thank very much! This is so great!
I have been searching a ton for this.
Thank you for the detailed and very quick response.

**Reply by geoffreymcgill** [reply]
You can also include specific lines from the file. The following sample demonstrates including just lines `10-20`. 

```md
{{ include "warning#10-20" }}

{{ include "/helpers/warning.md#10-20" }}
```

Hope this helps.
